packer/tools-setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+sudo debuginfo-install -q -y kernel
```

Comment:
I think we need it for using tools like `perf` -- @rxin added it if I am not wrong